css/properties/user-modify.json

```diff
@@ -0,0 +1,73 @@
+{
+  "css": {
+    "properties": {
+      "user-modify": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-moz-user-modify",
```

Comment:
I’ve moved this to [`https://developer.mozilla.org/docs/Web/CSS/user-modify`](https://developer.mozilla.org/docs/Web/CSS/user-modify)